src/core/component.js

```diff
@@ -136,6 +126,38 @@ Component.prototype = {
     return utils.extend({}, data);
   },
 
+  updateCachedValue: function (value, isSinglePropSchema) {
+    var parsedValue;
+    if (value === '') {
+      this.attrValue = undefined;
+      return;
+    }
+    if (typeof value === 'string') {
+      if (isSingleProp(this.schema)) {
+        parsedValue = this.schema.parse(value);
+        // To avoid bogus double parsings. Like for example
+        // when parsing a src id to it's url #monster -> models/monster.dae
+        if (typeof parsedValue === 'string') { parsedValue = value; }
```

Comment:
Can you explain how this avoids double parsing? Like with the `src type: `url(monster.dae)` -> `monster.dae` -> *tries to do a selector*. It every case, it will be a string.